app/Helpers/Helpers.php

```diff
@@ -189,16 +205,28 @@ function shouldUseBibleisBackwardCompat($key)
     if (isBibleisOrGideon($key)) {
```

Comment:
need to consider different values for deprecation_version, one for bibleis and one for gideons